tests/acceptance/volta_bypass.rs

```diff
@@ -15,19 +15,10 @@ fn shim_skips_platform_checks_on_bypass() {
         )
         .build();
 
-    #[cfg(unix)]
     assert_that!(
         s.process(shim_exe()),
         execs()
             .with_status(ExitCode::ExecutionFailure as i32)
             .with_stderr_contains("VOLTA_BYPASS is enabled[..]")
     );
-
-    #[cfg(windows)]
-    assert_that!(
-        s.process(shim_exe()),
-        execs()
-            .with_status(ExitCode::UnknownError as i32)
-            .with_stderr_contains("[..]is not recognized as an internal or external command[..]")
-    );
```

Comment:
Why is this test no longer valid? (To be extra clear, I am not saying you’re wrong about the change, it just is not at all obvious to me why it is being made!)

Reply:
I actually think this isn't that the test is invalid, but rather that the Windows behavior now accurately matches the Unix behavior, so they don't need to have separate expectations.

Reply:
@chriskrycho, @charlespierce was right on this.

Reply:
Excellent – thank you!